data/map.txt

```diff
@@ -30820,6 +35117,15 @@ planet Osaeli
 	landscape land/fields15
 	description `An Earth-like planet with a large biodiversity, at first look Osaeli appears to be an idyllic world where no lifeform ever became the predominant one; but looking more closely, the signs of an ancient devastation can still be seen. In the middle of the forests, huge amounts of debris are scattered in large areas; scans reveal many underground bunkers, partly destroyed and filled with water.`
 
+planet "Oup Je"
+	landscape land/canyon4
+	description `The many beveling formations here provide the Gegno Scin with a rich amount of materials for their cities and ships. Below are small woodlands that seem to be stripped down one way or another, with few living areas scattered about. It is apparent that this planet has been worn down over time, both to natural and unnatural causes.`
+	spaceport `The Scin here have different gear than those leaving and entering the spaceport, suggesting they have a particular place on the surface to be - most likely in the mines. They seem quite interested in you compared to other Gegno being that a majority of their days involve seeing mostly materials and machinery.`
```

Comment:
```suggestion
	spaceport `The Scin here have different gear than those leaving and entering the spaceport, suggesting they have a particular place on the surface to be - most likely in the mines. They seem quite interested in you compared to other Gegno, probably because a majority of their days involve seeing mostly minerals and machinery.`
```